include/onnxruntime/core/graph/graph_nodes.h

```diff
@@ -46,6 +47,14 @@ class ValidNodes {
     return cend();
   }
 
+  ConstReverseNodeIterator rbegin() const noexcept {
+    return {nodes_.crbegin(), nodes_.crend()};
+  }
```

Comment:
Are we sure the reverse of topological sort can be computed like this? I think we need to do inverse all edges and do an actual sort.